src/Compilers/Server/VBCSCompiler/BuildServerController.cs

```diff
@@ -63,12 +63,21 @@ protected virtual int RunServerCore(string pipeName, IClientConnectionHost conne
             return CommonCompiler.Succeeded;
         }
 
-        internal int RunServer(string pipeName, IClientConnectionHost clientConnectionHost = null, IDiagnosticListener listener = null, TimeSpan? keepAlive = null, CancellationToken cancellationToken = default(CancellationToken))
+        internal int RunServer(
+            string pipeName,
+            IClientConnectionHost clientConnectionHost = null,
+            string tempPath = null,
```

Comment:
Is there a reason we didn't just make this a required parameter? I'd be okay with a slightly bigger diff.